### ISSUE TITLE
conv_transpose1d with output padding backpropagation does not work

### ISSUE THREAD

**Issue by alexeib** [pytorch/pytorch#44949]
see the discussion here:
https://discuss.pytorch.org/t/loss-backward-failure-due-to-contiguous-issue/96416/9

in a nutshell, having several stacked conv_transpose1d layers with stride > 1 and output padding makes it not possible to backprop the loss with an error "ones need to be contiguous"

cc @ezyang @albanD @zou3519 @gqchen @pearu @nikitaved @ngimel

**Comment by ngimel** [comment]
Can you please provide a small runnable script reproducing the error?

**Comment by alexeib** [comment]
ok in my case the issue was that i had a layernorm after the transposed conv layers and was on pytorch 1.4. i found another issue that described a bug with layernorm following transposed conv and found that upgrading to pytorch 1.5+ fixed the issue

**Comment by lstrgar** [comment]
I'm having the same problem with a stack of conv transpose 1d / 2d and batchnorms, though I can't say specifically if the batchnorm has anything to do with it.
